object IntError 
  def foo(): Int(true) = { 42 }
end IntError